pkg/storkctl/clusterpair.go

```diff
@@ -285,7 +371,39 @@ func newCreateClusterPairCommand(cmdFactory Factory, ioStreams genericclioptions
 				util.CheckErr(err)
 				return
 			}
+
+			printMsg("Creating PX-Service Endpoint of DR cluster to create clusterpair...\n", ioStreams.Out)
```

Comment:
log should be Creating Secrets and BackupLocations ?

Reply:
Done